src/solver/conv_hip_implicit_gemm_grouped_wrw_xdlops.cpp

```diff
@@ -212,12 +227,55 @@ struct CKArgs
 };
 } // namespace
 
+template <int L, int H>
```

Comment:
all of these functions are strange. They don't quite (or just) do what the name suggests. I'd like these to be re-designed. Seems like there's a range (1, 128) where you either iterate linearly and loop around to `1` or you iterate in powers of `2`.